docs/http-api/swagger/authoritative-api-swagger.yaml

```diff
@@ -196,6 +196,10 @@ paths:
           in: query
           description: Limit output to the RRset of this type. Can only be used together with rrset_name.
           type: string
+        - name: include_disabled
+          in: query
+          description: '“true” or “false” (default), whether to include disabled RRsets in the response.'
```

Comment:
Given that `rrset_name`&type look like filters, I wonder if this should default to true. I even wonder if we should have the flag at all. API clients can filter out disabled for themselves. For the unfiltered output, they already had to.

* No flag, act like false: current behaviour which is the worst of all worlds.
* No flag, act like true: API clients need to learn to filter. Bunch of extra code for them.
* Flag, default true: API clients may need to start sending false.
* Flag, default false: new users will have a surprise the first time they use the filters.

"No flag, act like true" clearly is the right choice if this was a new API with new users. It is also the right choice for new users we grow after today. I think it might just be the right choice.

Reply:
I think they can, and older versions will ignore the flag (and act as 'false') so it's a quick change in a frontend/client lib. Meanwhile other users get less surprising behaviour.

Ok, let's go for

* we do have the flag
* it defaults to true